functionary/ui/views/teams/detail.py

```diff
@@ -1,33 +1,31 @@
 from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
 from django.views.generic.detail import DetailView
-from django.views.generic.list import ListView
 
-from core.auth import Permission
-from core.models import Team, TeamUserRole, Variable
-
-
-class TeamListView(LoginRequiredMixin, ListView):
-    model = Team
-
-    def get_queryset(self):
-        """Sorts based on team name, then env name."""
-        if self.request.user.is_superuser:
-            return super().get_queryset().order_by("name")
-        else:
-            teams = TeamUserRole.objects.filter(user=self.request.user).values(
-                "team_id"
-            )
-            return Team.objects.filter(id__in=[team["team_id"] for team in teams])
+from core.auth import ROLE_HEIRARCHY_MAP, Permission
+from core.models import Team, Variable
 
 
 class TeamDetailView(LoginRequiredMixin, UserPassesTestMixin, DetailView):
     model = Team
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        team = self.get_object()
+        team: Team = self.get_object()
+        users = [user_role for user_role in team.user_roles.all()]
```

Comment:
I think this came up before, but it's slightly confusing to have a variable named "users" that is not, in fact, a list of Users, but a list of TeamUserRoles.